<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>611</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QPushButton" name="Find">
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="4">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Save">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="Delete">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="Name_f"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="Name_s"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="Name_d"/>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Open">
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="Name_o"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
